src/handlers/store-raw-invoice-textract-data/store-expense-documents.ts

```diff
@@ -0,0 +1,43 @@
+import { SQSRecord } from "aws-lambda";
+import {
+  RAW_INVOICE_TEXTRACT_DATA_FOLDER_FAILURE,
+  RAW_INVOICE_TEXTRACT_DATA_FOLDER_SUCCESS,
+} from "../../shared/constants";
+import { fetchExpenseDocuments } from "./fetch-expense-documents";
+import { getQueuedExpenseAnalysisNotificationData } from "./get-queued-expense-analysis-notification-data";
+import { handleTextractFailure } from "./handle-textract-failure";
+import { handleTextractSuccess } from "./handle-textract-success";
+
+export async function storeExpenseDocuments(
+  record: SQSRecord,
+  destinationBucket: string
+): Promise<void> {
+  const { jobId, sourceBucket, sourceFileName, status } =
+    getQueuedExpenseAnalysisNotificationData(record);
+
+  const [sourceFolder] = sourceFileName.split("/");
+  if (
+    sourceFolder === RAW_INVOICE_TEXTRACT_DATA_FOLDER_FAILURE ||
+    sourceFolder === RAW_INVOICE_TEXTRACT_DATA_FOLDER_SUCCESS
+  )
+    return;
```

Comment:
I think maybe I don't understand what's meant to happen here, but it's not clear to me why we would just return no value in this case, and not log anything.

Reply:
Either of those would be fine, or else consider pulling the whole section of code into a helper method, e.g. alreadyProcessed().